app/course/[course_id]/manage/office-hours/settings/time-tracking/page.tsx

```
@@ -0,0 +1,430 @@
+  useEffect(() => {
+    if (courseId === null) {
+      router.replace("/");
+    }
+  }, [courseId, router]);
+
+  // Early return if course_id is invalid (prevents rendering with invalid data)
+  if (courseId === null) {
+    return null;
+  }
+
+  const allSessions = useHelpRequestWorkSessions();
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)